src/django_perf_rec/operation.py

```diff
@@ -0,0 +1,40 @@
+from django.conf import settings
+
+from django_perf_rec.utils import sorted_names
+
+
+class Operation(object):
```

Comment:
Python 3 doesn't need inheritance from `object`
```suggestion
class Operation:
```